terraform/app/modules/paas/variables.tf

```diff
@@ -215,4 +223,5 @@ locals {
   )
 
   database_name_suffix = var.add_database_name_suffix ? "${var.environment}" : null
+  app_name_suffix   = var.app_name == null ? var.environment : var.app_name
```

Comment:
app_name is never set so it will always be equal to var.environment. can we reomve app_name  and app_name_suffix?